scripts/firedrake-install

```diff
@@ -407,30 +422,33 @@ If you really want to use your own Python packages, please run again with the
 
 petsc_opts = "--download-eigen=%s/src/eigen-3.3.3.tgz " % firedrake_env
 
-petsc_opts += "--with-fortran-bindings=0 "
+full_opts = '--download-chaco --download-metis --download-parmetis \
+--download-scalapack --download-hypre --download-mumps --with-zlib --download-netcdf --download-pnetcdf --download-hdf5 --download-exodusii --with-fortran-bindings=0'
```

Comment:
Prefer set of full options here? that way adding is easier.

Do we want to add more additional LU solver types (e.g. superlue_dist, pastix, suitesparse [for umfpack]) by default?